### ISSUE TITLE
southpark.de - Videos don't load

### ISSUE THREAD

**Issue by johastl** [ryanbr/fanboy-adblock#1373]
### List the website(s) you're having issues:

`southpark.de`

### What happens?

Videos don't load

### List Subscriptions you're using:

(narrowed down to) Fanboy’s Annoyance

### Your settings

- OS/version: Win 7 Ultimate 64bit, patch level 06-2020
- Browser/version: Firefox 77.0.1 64bit
- Adblock Extension/version: 1.27.10

### Other details:

also tried in a VM:
- newly installed Win7 (default settings)
- Firefox v77.0.1 (default settings, no addons)
- uBlock v1.27.10  (default settings, no subscriptions)
-> **Videos on southpark.de are loading!**

- activated 'Fanboy’s Annoyance' list
-> **Videos on southpark.de are NOT loading!**

**Comment by peace2000** [comment]
Could this be related?: https://github.com/easylist/easylist/pull/5582

I cannot test because of geo restrictions and I'm too lazy to activate a proxy. :)

Does this whitelist rule help?:

`@@||c.evidon.com/sitenotice/evidon-sitenotice-tag.js$script,domain=southpark.de`

**Comment by johastl** [comment]
Yes, think it's the same as [easylist/easylist#5582](https://github.com/easylist/easylist/pull/5582)
`southparkstudios.nu` is geoblocked here and I also don't have a proxy set up,
but as it redirects to `southpark.de` I'm quite sure it's the same.

I added the rule you suggested to the whitelist
-> unfortunately it didn't help.

**Comment by johastl** [comment]
I played around a bit and I figured out, that it's this line in 'Fanboy’s Annoyance' filter list which causes the issue:
`/evidon-sitenotice-tag.`

For testing, I copied the entire 'Fanboy’s Annoyance' filter list to 'My filters'.
Then I added the domain exclusion `,domain=southpark.de` to the `/evidon-sitenotice-tag.` line and 
it worked!

So, my question is:
Is there a possibility to whitelist an already loaded line for a certain domain?
(that's probably what peace2000 tried already ...)
